variables.tf

```
@@ -40,6 +40,18 @@ variable "admin_user" {
   default     = "postgres"
 }
 
+variable "resource_proxy" {
+  type = object({
+    cpu     = number,
+    memory  = number
+  })
+  default = {
+    cpu         = 500,
+    memory      = 1024
+    description = "Postgres proxy resources"
+  }
+}
```

Could you move `description` out of `default` block
```suggestion
  }
  description = "Postgres proxy resources"
}
```

---

Should also be removed from the type object.

---

Should look like this:
```hcl
variable "resource_proxy" {
  type = object({
    cpu     = number,
    memory  = number,
  })
  description = "Postgres proxy resources"
  default = {
    cpu         = 500,
    memory      = 1024,
  }
}
```